src/docs/releases/3.0.0.md

```diff
@@ -142,6 +142,12 @@ There are lots of binary breaking changes in the Azure Search AI module most won
 
 - The permission `AzureAISearchPermissions.ManageAzureAISearchIndexes` was removed and the `AzureAISearchPermissions.ManageAzureAISearchISettings` was added.
 
+### Taxonomies Module
+
+Prior to this release, listing Taxonomies on the frontend used a slim pager (a pager with only "Next" and "Previous" buttons). With this release, the pager has been upgraded to a full-featured pager, allowing users to navigate through multiple pages directly.
```

Comment:
What should I as an app's maintainer keep an on, what can break? How to adapt?

Reply:
There should not be anything needed. The viewmodel is dynamic and should work as is.

Reply:
It won't work if in a shape override one does anything lower-level than rendering `Mode.Pager`. If you read or set any property of the object directly before, it'll break in runtime.

Before:
![{9DEAA9D3-FD1B-41EA-971F-1236950F1708}](https://github.com/user-attachments/assets/06cc8b8b-0610-49b0-a653-829b96eb314d)

After:

![{7A771426-2C5A-442C-ABE2-2CE202D138C0}](https://github.com/user-attachments/assets/22cb067a-bea0-4aa3-ba38-564efe642f74)

Reply:
@Piedone I updated the docs.